# polys_mapgen/config/mapgen.yaml
map_size_x: 20.0
map_size_y: 20.0
map_size_z: 5.0

map_origin_x: -10.0
map_origin_y: -10.0
map_origin_z: 0.0

